crypto/objects/obj_mac.num

```diff
@@ -1117,3 +1117,4 @@ RSA_SHA3_224		1116
 RSA_SHA3_256		1117
 RSA_SHA3_284		1118
 RSA_SHA3_512		1119
+scrypt		1120
```

Comment:
Is this left over from a previous version? On my system after making the changes to objects.txt and doing `make update` I get no changes in obj_mac.num